README.md

```diff
@@ -18,51 +18,12 @@ instead of [metrics](https://docs.rs/metrics/latest/metrics), and supports dynam
 
 ### Basic Usage
```

Comment:
I think we should have a very basic example in the README still. We can refer to examples for the other usage examples